client/src/utils/handleDates.js

```diff
@@ -0,0 +1,18 @@
+export function increaseDate(toAdd, from = new Date()) {
+  const updatedDate = from;
+  updatedDate.setDate(updatedDate.getDate() + (toAdd.days ?? 0));
+  updatedDate.setHours(updatedDate.getHours() + (toAdd.hours ?? 0));
+
+  return updatedDate;
+}
+
+export function setDateToNextHour(d, hours, minutes) {
```

Comment:
I'm not sure of what this function is doing, can you add some javadoc please? :smile:

Reply:
Yep yep!

Reply:
And just use a variable name a little more expressive than `d` 😀